srt_test.go

```diff
@@ -27,3 +27,22 @@ func TestSRT(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(c), w.String())
 }
+
+func TestSRTMissingSequence(t *testing.T) {
+	// Open
+	s, err := astisub.OpenFile("./testdata/missing-sequence-in.srt")
+	assert.NoError(t, err)
+	assertSubtitleItems(t, s)
+
+	// No subtitles to write
+	w := &bytes.Buffer{}
+	err = astisub.Subtitles{}.WriteToSRT(w)
+	assert.EqualError(t, err, astisub.ErrNoSubtitlesToWrite.Error())
+
+	// Write
+	c, err := ioutil.ReadFile("./testdata/missing-sequence-out.srt")
```

Comment:
I don't think we need to use another file here since it's the same output as `./testdata/example-out.srt` 🤔

Reply:
Great point, thanks. I'll remove it and use the existing output.